mmv1/third_party/terraform/website/docs/r/composer_user_workloads_secret.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Cloud Composer"
+description: |-
+  Kubernetes secret in Composer Environment workloads.
+---
+
+# google\_composer\_user\_workloads\_secret
+
+Provides a way to manage Kubernetes secret in Composer Environment workloads.
```

Comment:
Nit: secret -> "Secrets"

Reply:
Fixed in separate [PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/10490)